code/datums/elements/strippable.dm

```diff
@@ -74,6 +74,8 @@
 /// This should be used for checking if an item CAN be equipped.
 /// It should not perform the equipping itself.
 /datum/strippable_item/proc/try_equip(atom/source, obj/item/equipping, mob/user)
+	if(SEND_SIGNAL(user, COMSIG_CAN_STRIP, source, user, equipping) & COMPONENT_CANT_STRIP)
```

Comment:
This is a bad signal. Signals should say what's going on, not be a request. Change it to `COMSIG_TRY_STRIP` or similar, just like everything else.